device.go

```diff
@@ -55,8 +55,9 @@ type Device struct {
 	FirmwareVersion     string `json:"firmwareVersion,omitempty"`
 	SerialNumber        string `json:"serialNumber,omitempty"`
 	DisplayName         string `json:"displayName,omitempty"`
+	HardwareBoardType   int    `json:"hardwareBoardType,omitempty"`
```

Comment:
Nit: put this between product name and firmware build number to match the order of the JSON from the device.